src/main/java/ui/BoardAutoCreator.java

```diff
@@ -150,7 +152,9 @@ private void createSampleBoard() {
 
         triggerBoardSaveEventSequence(SAMPLE_BOARD);
 
-        DialogMessage.showInformationDialog("Auto-create Board - " + SAMPLE_BOARD, SAMPLE_BOARD_DIALOG);
+        if (isDialogShown){
+            DialogMessage.showInformationDialog("Auto-create Board - " + SAMPLE_BOARD, SAMPLE_BOARD_DIALOG);
+        }
```

Comment:
The dialog is needed when the sample board is created using `Auto Create` menu. The dialog is not seen by new users.